index.js

```diff
@@ -9,6 +9,11 @@ exports.functionThatReturnsFalse = require('./functionThatReturnsFalse');
 exports.functionThatReturnsNull = require('./functionThatReturnsNull');
 exports.functionThatReturnsThis = require('./functionThatReturnsThis');
 exports.functionThatReturnsArgument = require('./functionThatReturnsArgument');
+exports['boolean'] = require('./boolean');
```

Comment:
Right, but reserved words can still be used as key in ES5 and up.
Also, importing/requiring these will cause a bit of a nuisance since you can’t do `import {null} from 'empty'`.